resources/content/config-content/haproxy/content/etc/haproxy/haproxy.cfg.ftl

```diff
@@ -97,6 +97,7 @@ backend ${listener.uuid}_${backend.uuid}_backend
 </#list>
 </#if>
 
-listen default 
+listen default
+<#if lbHealthCheck??><#assign defaultPort = lbHealthCheck.port><#else><#assign defaultPort = 42></#if>
```

Comment:
I don't understand what your doing here?  You assign defaultPort but don't seem to use it.